zerologger/generate.go

```diff
@@ -131,21 +130,27 @@ func updateExampleFile(genpkg string, root *expr.RootExpr, f *fileToModify) {
 			s.Source = strings.Replace(s.Source, `logger = log.New(os.Stderr, "[{{ .APIPkg }}] ", log.Ltime)`,
 				`logger = log.New("{{ .APIPkg }}", false)`, 1)
 			s.Source = strings.Replace(s.Source, "adapter = middleware.NewLogger(logger)", "adapter = logger", 1)
+			s.Source = strings.Replace(s.Source, "handler = httpmdlwr.Log(adapter)(handler)", "handler = log.ZerodriverHttpMiddleware(adapter)(handler)", 1)
 			s.Source = strings.Replace(s.Source, "handler = httpmdlwr.RequestID()(handler)",
 				`handler = httpmdlwr.PopulateRequestContext()(handler)
-				handler = log.ZerodriverHttpMiddleware(logger)(handler)
 				handler = httpmdlwr.RequestID(httpmdlwr.UseXRequestIDHeaderOption(true))(handler)`, 1)
 			s.Source = strings.Replace(s.Source, `logger.Printf("[%s] ERROR: %s", id, err.Error())`,
 				`logger.Error().Str("id", id).Err(err).Send()`, 1)
 			s.Source = strings.Replace(s.Source, "logger.Print(", "logger.Info().Msg(", -1)
 			s.Source = strings.Replace(s.Source, "logger.Printf(", "logger.Info().Msgf(", -1)
 			s.Source = strings.Replace(s.Source, "logger.Println(", "logger.Info().Msg(", -1)
+			s.Source = strings.Replace(s.Source, "logger.Fatal(", "logger.Fatal().Msg(", -1)
```

Comment:
goa uses logger.Fatal instead of logger.Print & os.Exit https://github.com/goadesign/goa/pull/3081

Reply:
Thanks! Agreed.